README.md

```diff
@@ -1,7 +1,27 @@
 # Hyperworld
+[![image](https://github.com/millennIumAMbiguity/hyperworld/assets/37588844/d5073b5f-2d76-4ef0-8276-61d05f8e8f64)](https://youtu.be/Fkyq9AxCcBs?si=RGCMwWRw0cUjR2bO)
```

Comment:
Currently, this image points to a github asset in the `millennIumAMbiguity/hyperworld` repo, which I believe could inadvertently add an unexpected dependency. I believe a simple textual link like "View demo on YouTube" or something like that would be good here. However, if you feel strongly that it should be an image, I can try to set it up so that the link points to the patowen repo instead.

Reply:
Usually, I would use Imgur, but after they delete old pics now I no longer do that.
here I have a `patowen/hyperworld/assets` link.
![304918695-d5073b5f-2d76-4ef0-8276-61d05f8e8f64](https://github.com/patowen/hyperworld/assets/37588844/97efb41a-4ae8-4c1d-a4bb-3c935bf2614c)